src/yaml/bsc/vdk-prop-pack-vol03-gevo-engines.yaml

```diff
@@ -0,0 +1,20 @@
+group: bsc
+name: vdk-gevo-props
+version: "1.0"
+subfolder: 100-props-textures
+info:
+  summary: VDK GEVo Props by Vester
```

Comment:
Yes you're right, I'll let the props depend on the textures.

It took me a good while indeed to figure out how this package was meant to be used. Another example of where sc4pac can come in really handy.

Also, it's absolutely insane that there are 140 texture variants to choose from, and this 5 times, lol. I simply included them all, but it's a bit strange when you install this with the gui - as typically this is a dependency for another package - and suddenly you get hit with having to choose 1 of 140 options, 5 times in a row. Perhaps that we could just pick a few randomly as well, it doesn't matter that much anyway I guess. Let me know what you think.

Reply:
Yeah, I guess you're right. We could just pick 5 skins. Users can override this by doing a manual install if desired. Do you have any preferences for the skins? Otherwise I'll just pick some.

Reply:
Not at all, you can just pick some.